spec/errorSpec.js

```diff
@@ -1,6 +1,5 @@
 var webdriver = require('selenium-webdriver');
 var common = require('./common.js');
-require('../index.js');
 
```

Comment:
Though it does make me wonder if we now need to include an 'already initialized' failure if someone does it twice. Previously, this wasn't a problem since node `require` caching would prevent it. What do you think?